hub/package-manager/winget/pinning.md

```diff
@@ -0,0 +1,235 @@
+---
+title: The winget pin command
+description: Use the winget pin command and subcommands to manage package pins. Pins can limit the Windows Package Manager from upgrading certain packages completely or within a specific range.
+ms.date: 06/16/2023
+ms.topic: reference
+ms.localizationpriority: medium
+ms.custom: kr2b-contr-experiment
+---
+
+# The winget pin command
+
+The [winget](index.md) **pin** command allows you to limit the Windows Package Manager from upgrading a package to specific ranges of versions, or it can prevent it from upgrading a package altogether. A pinned package may still upgrade on its own and be upgraded from outside the Windows Package Manager. 
+
+## Pin Types
+WinGet supports three types of Package Pins:
+- **Pinning**:
+The package is excluded from `'winget upgrade --all'` but allows `'winget upgrade <package>'`. You can use the `'–-include-pinned'` argument to let `'winget upgrade --all'` include pinned packages.
```

Comment:
```suggestion
The package is excluded from `'winget upgrade --all'` but allows `'winget upgrade <package>'`. You can use the `'--include-pinned'` argument to let `'winget upgrade --all'` include pinned packages.
```
The character used is a unicode character which when copy-pasted into the terminal wouldn't be recognized as a valid option